src/cryptoadvance/specterext/spectrum/server_endpoints/templates/spectrum/components/spectrum_info.jinja

```diff
@@ -0,0 +1,85 @@
+<h1>{{ _("Bitcoin Core Node Info") }}:</h1>
+{% if specter.chain %}
+    {% if specter.info['pruned'] %}
+        <p class="warning"><img src="{{ url_for('static', filename='img/info_sign.svg') }}" style="width: 20px;"/><br>{{ _("You are using a pruned node.")}}<br>{{ _("Pruned nodes may not be able to scan for old balances when importing an existing wallet.")}}<br>{{ _("It is recommended to use Specter with an unpruned node.") }}</p>
+        <br>
+    {% elif not specter.info['blockfilterindex'] %}
+        <p class="warning"><img src="{{ url_for('static', filename='img/info_sign.svg') }}" style="width: 20px;"/><br>{{ _("Your node does not have ")}}<code>blockfilterindex</code>{{ _(" enabled.")}}<br>{{ _("Setting ")}}<code>blockfilterindex=1</code>{{ _(" in your ")}}<code>bitcoin.conf</code>{{ _(" file is recommended as it takes just a few GB of storage")}}<br>{{ _("and helps to speed-up blockchain rescanning.") }}</p>
+        <br>
+    {% endif %}
+    Connect Your own Node!!!
```

Comment:
let's audit what this text should actually be. we should not use `!!!` in our copy. is this driving the user to do some action? Should it be "[Warning Icon] Connecting to public electrum servers may expose some information including your ip address and transaction history."